### PR TITLE
Formatting section added

### LINKED ISSUE #971: Message formatting options are undocumented
It would be nice to have someone document message formatting options somewhere - ideally in the user guide (either within [messaging ](https://rocket.chat/docs/user-guides/messaging/) or as a stand-alone page).  Web searches mostly turn up information on how GitHub or SourceForge format documentation for contributions to the product. :)

Originally opened at RocketChat/Rocket.Chat#10478, but then not followed-up upon for some reason.

### PR DESCRIPTION
**Pull request by Davide95**
Closes #971
Closes RocketChat/Rocket.Chat#10478

### CODE REVIEW

File: user-guides/messaging/README.md [hunk 1]
```diff
@@ -108,6 +108,15 @@ Graphical emojis can be enabled/disabled in [Your Account](../managing-your-acco
...
 
+## Formatting
+
+To format properly your message you can wrap the text you want to augment with special characters:
+
+* Bold: &#42;Lorem ipsum dolor&#42;;
+* Italic: &#95;Lorem ipsum dolor&#95;;
+* Strike: &#126;Lorem ipsum dolor&#126;;
+* Inline code: &#96;Lorem ipsum dolor&#96;;
+
```
Comment (knrt10) [inline user-guides/messaging/README.md:121]: You should also add information about link formatting too. Example there are 3 ways.

1. [Text]&#40;http://link)
2. ![alt text]&#40;http://image url)
3. &#60;http://link|Text>

**Review by knrt10 — CHANGES_REQUESTED** [review]

File: user-guides/messaging/README.md (hunk 1 above)
Comment (Davide95) [reply user-guides/messaging/README.md:121]: Updated.
I've inserted the first one and the third, but I'm not sure about the second.

e.g. `Alt text doesn't work on the webapp.

Comment (knrt10) [reply user-guides/messaging/README.md:121]: no worries, issue has been opened for it. Once others acknowledge it, fix will be made. See here [issue13606](https://github.com/RocketChat/Rocket.Chat/issues/13606).

Comment (Davide95) [reply user-guides/messaging/README.md:121]: Done

**Review by MartinSchoeler — APPROVED** [review]

File: user-guides/messaging/README.md (hunk 1 above)
Comment (themantheycallcris) [reply user-guides/messaging/README.md:121]: Can someone add how to change the font size for a message?   Sorry if this is the wrong place to ask, I'm very new to this.

**Review by themantheycallcris — COMMENTED** [review]
Can someone add to the README.md how to change the font size for a message in RocketChat? Sorry if this is the wrong place to ask, I'm very new to this.

**Pull request merged**